translate/filters/checks.py

```diff
@@ -862,7 +862,13 @@ def puncspacing(self, str1, str2):
             return True
 
         str2 = self.filteraccelerators(self.filtervariables(str2))
+        # Substitute: nbsp
         str2 = str2.replace(u"\u00a0", u" ")
+        # Strip: Bidi markers and ZW* chars
+        str2 = str2.translate(
+            {ord(c): None for c in
+             u"\u200e\u200f\u202b\u202a\u202e\u202d\u202c\u2069\u2068\u2067\u2066" +
+             u"\u200d\u200c"})
```

Comment:
Nit: Instead of a string I would have used a tuple of characters, one character per line with a comment besides with the name for the character, like in https://github.com/translate/translate/blob/master/translate/lang/ar.py#L51-L62